src/main/groovy/com/bmuschko/gradle/docker/tasks/network/DockerCreateNetwork.groovy

```diff
@@ -35,4 +62,36 @@ class DockerCreateNetwork extends AbstractDockerRemoteApiTask {
         networkId.set(createdNetworkId)
         logger.quiet "Created network with ID '$createdNetworkId'."
     }
+
+
+    static class Ipam {
+        private static final String SUBNET = "subnet"
+
+        @Input
+        @Optional
+        ListProperty<Map> config
```

Comment:
This should be a List of complex objects, in this case `Config`. A Map is too generic. I can touch this up on my end.